src/components/App.js

```diff
@@ -1,8 +1,11 @@
 import React, { Component } from 'react';
 
+import Menu from './Menu';
 import TodoList from './TodoList';
 import TodoForm from './TodoForm';
-import Menu from './Menu';
+import SearchForm from './SearchForm';
+import { purifyText } from '../utils/utils';
```

Comment:
We should change the name to `utils/string` or something similar. `utils/utils` is redundant.